tests/data/GlyphsFileFormatv3.diff

```diff
@@ -0,0 +1,1676 @@
+  }
+  );
+  name = Regular;
+  numberValues = (
+  123
+  );
+  stemValues = (
+  123,
+  321,
+  234
+  );
+  userData = {
+  "Some Key" = "Some Value";
+  };
+  visible = 1;
+  },
+  {
+  axesValues = (
+  900
+  );
+  customParameters = (
+  {
+  name = "Default Layer Width";
+  value = "600";
+  }
+  );
+  guides = (
+  {
+  angle = 24.582;
+  lockAngle = 1;
+  pos = (192,216);
+  showMeasurement = 1;
+  }
+  );
+  iconName = Bold;
+  id = "C2ECF50A-02EF-4989-A14C-AF8E838D1105";
+  metricValues = (
+  {
+  over = 15;
+  pos = 800;
+  },
+  {
+  over = 15;
+  pos = 700;
+  },
+  {
+  over = 15;
+  pos = 500;
+  },
+  {
+  over = -15;
+  },
+  {
+  over = -15;
+  pos = -200;
+  },
+  {
+  over = 15;
+  pos = 550;
+  },
+  {
+  over = 12;
+  pos = 123;
+  }
+  );
+  name = Black;
+  numberValues = (
+  123
+  );
+  stemValues = (
+  123,
+  321,
+  234
+  );
+  }
+  );
+  glyphs = (
+  {
+  color = 7;
+  glyphname = A;
+- kernBottom = A;
+  kernLeft = A;
+  kernRight = A;
+- kernTop = A;
+  lastChange = "2023-02-25 14:46:49 +0000";
+  layers = (
+  {
+  anchors = (
+  {
+  name = bottom;
+  pos = (230,0);
+  },
+  {
+  name = ogonek;
+  pos = (413,10);
+  },
+  {
+  name = top;
+  pos = (230,700);
+- userData = {
+- "Some Key" = "Some Value";
+- };
+  },
+  {
+  name = top.alt;
+  pos = (10,635);
+  }
+  );
+  color = 0;
+  guides = (
+  {
+  angle = 8.1446;
+  lockAngle = 1;
+  locked = 1;
+  pos = (151,402);
+  showMeasurement = 1;
+  },
+  {
+  name = "guide name";
+  pos = (186,310);
+  },
+  {
+- orientation = right;
+  angle = 90;
+  pos = (-74,271);
+  }
+  );
+  hints = (
+  {
+  horizontal = 1;
+  origin = (0,0);
+  target = (0,7);
+  type = Stem;
+  },
+  {
+  name = _corner.cut;
+  options = 1;
+  origin = (0,7);
+  scale = (1.1,0.9);
+  type = Corner;
+  }
+  );
+  layerId = m01;
+  metricRight = "=20";
+  shapes = (
+  {
+  closed = 1;
+  nodes = (
+  (10,66,l),
+  (92,-21,o),
+  (338,-35,o),
+  (439,66,cs),
+  (578,205,o),
+  (522,462,o),
+  (439,608,c),
+  (10,608,l,{
+  name = "Hallo	Welt";
+  test = "Hallo
+  Welt";
+  })
+  );
+  }
+  );
+  userData = {
+  "Some Key" = "Some Value";
+  };
+  width = 459;
+  },
+  {
+  associatedMasterId = m01;
+  attr = {
+  color = 1;
+  };
+  layerId = "B53B276E-7ED6-4F56-94FF-4162BC3B585A";
+  shapes = (
+  {
+  attr = {
+  strokeColor = (0,84);
+  strokeWidth = 21;
+  };
+  closed = 0;
+  nodes = (
+  (20,35,l),
+  (91,35,o),
+  (97,35,o),
+  (100,5,c),
+  (103,35,o),
+  (109,35,o),
+  (180,35,c)
+  );
+  }
+  );
+  width = 459;
+  },
+  {
+  anchors = (
+  {
+  name = bottom;
+  pos = (230,0);
+  },
+  {
+  name = ogonek;
+  pos = (413,10);
+  },
+  {
+  name = top;
+  pos = (230,700);
+  }
+  );
+  guides = (
+  {
+  angle = 8.1446;
+  lockAngle = 1;
+  locked = 1;
+  pos = (151,402);
+  showMeasurement = 1;
+  }
+  );
+  hints = (
+  {
+  horizontal = 1;
+  origin = (0,0);
+  target = (0,7);
+  type = Stem;
+  },
+  {
+  horizontal = 1;
+- place = (-46, 58);
++ place = (-46,58);
```

Comment:
Note sure why Glyphs produces it without spaces. @schriftgestalt, would you be able to assist? Should be be with or without spaces when formatting points? Thank you!

Reply:
From the Spec:
> Some lists (enclosed in normal parenthesis) are put on one line (e.g., **(100,200)**). This is to make it easier to read and save space. We call it tuple for now.
> - this is used for colors, points and nodes

This shows it without spaces.

Reply:
And it is a bit tricky to follow all those comments. Can we find a better way to talk about this? Maybe we can do a zoom (or whatever service you prefer)?

Reply:
Thank you for the answer. This is exactly my point that, according to the spec, it should be without spaces, which is what glyphsLib produces. However, it is with spaces in the original file. (The red chunks are what removed by glyphsLib relative to `GlyphsFileFormatv3.glyphs`, and the green ones are what has been added by glyphsLib.)

Here it is:

https://github.com/schriftgestalt/GlyphsSDK/blob/8639eaf79645e664be06e001fe0417cc74689da7/GlyphsFileFormat/GlyphsFileFormatv3.glyphs#L376

So I am not sure what is happening here. Maybe `place` is of a different type?

Reply:
> Can we find a better way to talk about this? Maybe we can do a zoom (or whatever service you prefer)?

Yes, I would be happy to have a call to discuss if you prefer.

Reply:
I’m on CET. I’m generally available between 9am to 17pm.

Reply:
I am in the same time zone. Please let me know what it would be convenient for you, and I will schedule a call.

Reply:
Thank you for the call earlier today!

@anthrotype, to keep you updated, the above is due to a typo in Glyphs. The behavior exhibited by glyphsLib here is correct. So no action needed. The reference file will be updated accordingly.